.github/workflows/project-checks.yaml

```diff
@@ -0,0 +1,55 @@
+name: Project Checks
+
+on:
+  push:
+    branches:
+      - main
+      - "release/**"
+      - "test/**"
+  pull_request:
+    branches:
+      - main
+      - "release/**"
+
+env:
+  GO_VERSION: "1.19.5"
+
+jobs:
+  checkout:
+    name: Checkout
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
+    steps:
+      - name: Set env
+        shell: bash
+        run: |
+          echo "GOPATH=${{ github.workspace }}" >> $GITHUB_ENV
+          echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
+      - uses: actions/checkout@v3
+        with:
+          path: src/github.com/containerd/nri-plugin-collection
+          fetch-depth: 25
+
+  verify:
+    name: Verify
+    needs: [checkout]
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - uses: actions/checkout@v3
+        with:
+          path: src/github.com/containerd/nri-plugin-collection
```

Comment:
As we do not have this yet, should we keep this as Draft until we have the plugins somewhere proper?

Reply:
i think this is a relative(local) path to where we clone the current project

Reply:
Yes, that's just our configured path where the checkout goes. I was a bit jumping ahead of us to see how that repo path would look like. It can be changed.